scheduler/config/rbac/leader_election_role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: leader-election-role
+  name: keptn-scheduler-leader-election-role
```

Comment:
up to you, but to be consistent we could also add a specific prefix for the role in the lifecycle-operator

Reply:
you mean like `keptn-lifecycle-operator-leader-election-role` ?